migrations/20200807083335-add-collectives-data-use-custom-host-fee.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  /**
+   * Set "Collectives" -> "data" -> "useCustomHostFee" to true when their hostFeePercent
+   * is different from the one of the host.
+   */
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(`
+      UPDATE ONLY "Collectives" c
+      SET
+        data = (
+          CASE WHEN c.data IS NULL 
+          THEN '{"useCustomHostFee": true}'::jsonb
+          ELSE c.data::jsonb || '{"useCustomHostFee": true}'::jsonb
```

Comment:
Is this else condition equivalent to this?
```
jsonb_set(data, '{useCustomHostFee}', 'true')
```

Reply:
Does Jsonb_set creates the Jason object if the column is null ?

Reply:
I don't think it does. But I'm referring just to the ELSE condition here.